Cargo.toml

```diff
@@ -367,7 +367,10 @@ grpcio-health = { version = "0.10.4", default-features = false, features = ["pro
 tipb = { git = "https://github.com/pingcap/tipb.git" }
 kvproto = { git = "https://github.com/pingcap/kvproto.git" }
 yatp = { git = "https://github.com/tikv/yatp.git", branch = "master" }
-tokio-timer = { git = "https://github.com/tikv/tokio", branch = "tokio-timer-hotfix" }
+# tokio-timer = { git = "https://github.com/tikv/tokio", branch = "tokio-timer-hotfix" }
+# TODO: after the `upgrade_crossbeam_utils` has been merged, the `tokio-timer` should revert to the original.
+tokio-timer = { git = "https://github.com/LykxSassinator/tokio-timer", branch = "upgrade_crossbeam_utils" }
```

Comment:
Make sure it's updated before the merge.

Reply:
yep.